caper/caper_init.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
```

Comment:
It doesn't make sense to leave the shebang in here if this is consumed as a module.

Reply:
good point. will remove it

Reply:
sorry, will remove those shebangs in the latest PR.